scheduler/generic_sched_test.go

```diff
@@ -6379,3 +6379,147 @@ func TestPropagateTaskState(t *testing.T) {
 		})
 	}
 }
+
+// Tests that a client disconnect generates attribute updates and follow up evals.
+func TestServiceSched_Client_Disconnect_Creates_Updates_and_Evals(t *testing.T) {
+	cases := []struct {
+		maxClientDisconnect time.Duration
+		rescheduled         bool
+		count               int
+	}{
+		{
+			maxClientDisconnect: 10 * time.Minute,
+			rescheduled:         true,
+			count:               1,
+		},
+	}
+
+	for _, tc := range cases {
+		t.Run(fmt.Sprintf(""), func(t *testing.T) {
```

Comment:
If we're going to do table tests, provide a name so that it gives us readable test output:

```suggestion
		t.Run(tc.name, func(t *testing.T) {
```

There's currently only one test here though and the test assertions look pretty complex. Maybe not worth having a table?

Reply:
I'm hoping to add more in later rounds. I can delete this table, but then I will just have to add it back later as I flesh out tests. Is it ok to leave as is for now?

Reply:
Removing until I can find a way to table test.